xrechnung/xrechnung-2_2_0.xml.twig

```diff
@@ -0,0 +1,145 @@
, 'Kamerun': 'CM', 'Kanada': 'CA', 'Kap Verde': 'CV', 'Karibische Niederlande': 'BQ', 'Kasachstan': 'KZ', 'Kenia': 'KE', 'Kirgisistan': 'KG', 'Kiribati': 'KI', 'Kokosinseln': 'CC', 'Kolumbien': 'CO', 'Komoren': 'KM', 'Kongo': 'CG', 'Kroatien': 'HR', 'Kuba': 'CU', 'Kuwait': 'KW', 'Laos': 'LA', 'Lesotho': 'LS', 'Lettland': 'LV', 'Libanon': 'LB', 'Liberia': 'LR', 'Libyen': 'LY', 'Liechtenstein': 'LI', 'Litauen': 'LT', 'Luxemburg': 'LU', 'Macao': 'MO', 'Madagaskar': 'MG', 'Malawi': 'MW', 'Malaysia': 'MY', 'Malediven': 'MV', 'Mali': 'ML', 'Malta': 'MT', 'Marianen': 'MP', 'Marokko': 'MA', 'Marshall Inseln': 'MH', 'Martinique': 'MQ', 'Mauretanien': 'MR', 'Mauritius': 'MU', 'Mayotte': 'YT', 'Mazedonien': 'MK', 'Mexiko': 'MX', 'Mikronesien': 'FM', 'Mocambique': 'MZ', 'Moldavien': 'MD', 'Monaco': 'MC', 'Mongolei': 'MN', 'Montenegro': 'ME', 'Montserrat': 'MS', 'Namibia': 'NA', 'Nauru': 'NR', 'Nepal': 'NP', 'Neukaledonien': 'NC', 'Neuseeland': 'NZ', 'Nicaragua': 'NI', 'Niederlande': 'NL', 'Niger': 'NE', 'Nigeria': 'NG', 'Niue': 'NU', 'Nord Korea': 'KP', 'Norfolk Inseln': 'NF', 'Norwegen': 'NO', 'Oman': 'OM', 'Österreich': 'AT', 'Osttimor': 'TL', 'Pakistan': 'PK', 'Palästina': 'PS', 'Palau': 'PW', 'Panama': 'PA', 'Papua Neuguinea': 'PG', 'Paraguay': 'PY', 'Peru': 'PE', 'Philippinen': 'PH', 'Pitcairn': 'PN', 'Polen': 'PL', 'Portugal': 'PT', 'Puerto Rico': 'PR', 'Qatar': 'QA', 'Reunion': 'RE', 'Ruanda': 'RW', 'Rumänien': 'RO', 'Russland': 'RU', 'Saint Lucia': 'LC', 'Saint-Barthélemy': 'BL', 'Saint-Martin': 'MF', 'Sambia': 'ZM', 'Samoa': 'AS', 'Samoa': 'WS', 'San Marino': 'SM', 'Sao Tome': 'ST', 'Saudi Arabien': 'SA', 'Schweden': 'SE', 'Schweiz': 'CH', 'Senegal': 'SN', 'Serbien': 'RS', 'Seychellen': 'SC', 'Sierra Leone': 'SL', 'Singapur': 'SG', 'Sint Maarten': 'SX', 'Slowakei': 'SK', 'Slowenien': 'SI', 'Solomon Inseln': 'SB', 'Somalia': 'SO', 'Spanien': 'ES', 'Sri Lanka': 'LK', 'St. Helena': 'SH', 'St. Kitts Nevis Anguilla': 'KN', 'St. Pierre und Miquelon': 'PM', 'St. Vincent': 'VC', 'Süd Korea': 'KR', 'Südafrika': 'ZA', 'Sudan': 'SD', 'Südgeorgien und die Südlichen Sandwichinseln': 'GS', 'Südsudan': 'SS', 'Surinam': 'SR', 'Svalbard und Jan Mayen Islands': 'SJ', 'Eswatini': 'SZ', 'Syrien': 'SY', 'Tadschikistan': 'TJ', 'Taiwan': 'TW', 'Tansania': 'TZ', 'Thailand': 'TH', 'Togo': 'TG', 'Tokelau': 'TK', 'Tonga': 'TO', 'Trinidad Tobago': 'TT', 'Tschad': 'TD', 'Tschechische Republik': 'CZ', 'Tunesien': 'TN', 'Türkei': 'TR', 'Turkmenistan': 'TM', 'Turks und Kaikos Inseln': 'TC', 'Tuvalu': 'TV', 'Uganda': 'UG', 'Ukraine': 'UA', 'Ungarn': 'HU', 'United States Minor Outlying Islands': 'UM', 'Uruguay': 'UY', 'Usbekistan': 'UZ', 'Vanuatu': 'VU', 'Vatikan': 'VA', 'Venezuela': 'VE', 'Vereinigte Arabische Emirate': 'AE', 'Vereinigte Staaten von Amerika': 'US', 'Vietnam': 'VN', 'Virgin Island (Brit.)': 'VG', 'Virgin Island (USA)': 'VI', 'Wallis et Futuna': 'WF', 'Weissrussland': 'BY', 'Westsahara': 'EH', 'Zentralafrikanische Republik': 'CF', 'Zimbabwe': 'ZW', 'Zypern': 'CY'} %}
```

Comment:
do we need that, doesn't seem to be used

Reply:
It's a left over. I'll remove it.